apple/Handlers/RNPanHandler.m

```diff
@@ -78,6 +86,24 @@ - (void)setMinimumNumberOfTouches:(NSUInteger)minimumNumberOfTouches
 {
   _realMinimumNumberOfTouches = minimumNumberOfTouches;
 }
+
+- (void)tryUpdateStylusData:(UIEvent *)event
+{
+  UITouch *touch = [[event allTouches] anyObject];
+
+  if (touch.type != UITouchTypePencil) {
+    return;
+  }
+
+  _stylusData.altitudeAngle = touch.altitudeAngle;
+  _stylusData.azimuthAngle = [touch azimuthAngleInView:nil];
+  _stylusData.pressure = touch.force / touch.maximumPossibleForce;
+
+  Tilt tilts = spherical2tilt(_stylusData.altitudeAngle, _stylusData.azimuthAngle);
+
+  _stylusData.tiltX = tilts.tiltX;
+  _stylusData.tiltY = tilts.tiltY;
```

Comment:
Why not keep the struct in StylusData?

Reply:
I wanted to keep it similar across all platforms, though I can change that if you prefer to 😓

Reply:
Nah, it just looked a bit funny to me